models/UserModel.js

```diff
@@ -14,7 +14,29 @@ const updateUserInfo = `
   RETURNING "userId", "displayName", "username";
 `;
 
+const getUsers = `
+  SELECT 
+      u."userId",
+      u."displayName",
+      u.username
+  FROM 
+      public.app_user u
+  LEFT JOIN 
+      public.follow f ON u."userId" = f."followedUserId"
+  WHERE 
+      u.username ILIKE '%' || $1 || '%'
+      OR u."displayName" ILIKE '%' || $1 || '%'
```

Comment:
This is a matter of preference, but another option is to add the wildcards directly to the input value in the controller. For example, we could do something like this in the controller:

// UserController.js
```
const { keyword } = req.query;
const searchQuery = `%${keyword }%`
const query = await pool.query(userQueries.getUsers, [searchQuery])
```

Then we can change the query to the following instead
```suggestion
  WHERE 
      u.username ILIKE $1'
      OR u."displayName" $1'
```

I don't think it makes any difference functionally, it's just up to us to decide which we think reads easier. Let me know what you think

Reply:
I like this alot better, the readability should be simple inside of our queries and this helps makes it look more refined.